### DISCUSSION TITLE
vercel部署访问报错

### DISCUSSION THREAD

**Discussion by stacklix** [walinejs/waline#1555] [Q&A] [answered]
使用waline最新代码在vercel进行部署，github作为数据库，访问时出现截图所示的异常
![image](https://user-images.githubusercontent.com/5688532/199006114-9d58a131-98db-43a1-8be0-bbdd9e7f35f8.png)

请教各位大佬，这是哪里的配置有问题吗

**Comment by lizheming** [comment, marked as answer]
不建议使用 GitHub CSV 存储。建议你换其他存储服务。

**Reply by leimao** [reply]
我似乎也碰到了这个问题。如果 GitHub CSV 不再支持，就应该在文档中说明。

---
I seem to have run into this problem too. If GitHub CSV is no longer supported, this should be stated in the documentation.
